src/Microsoft.Health.Fhir.Core/Messages/BulkImport/ImportBundleRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Features.Operations.Import;
+using Microsoft.Health.Fhir.Core.Features.Operations.Import.Models;
+using Microsoft.Health.Fhir.Core.Features.Persistence;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Import
+{
+    public class ImportBundleRequest : IRequest<ImportBundleResponse>
+    {
+        public ImportBundleRequest(IList<ImportResource> resources)
+        {
+            Resources = resources;
```

Comment:
We should add a validation and check if resources are not null.

Reply:
This is not possible in current code. Why do we need to do this?

Reply:
Done.